src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -175,9 +173,10 @@
                           @(subscribe [:state-query
                                        (cond-> [:application :answers id]
                                                idx (concat [:values idx 0]))]))
-            on-change   (cond @verify-email? (partial email-verify-field-change field-descriptor answer)
-                              idx            (partial multi-value-field-change field-descriptor 0 idx)
-                              :else          (partial textual-field-change field-descriptor))
+            on-change   (fn [evt]
+                          (cond @verify-email? (email-verify-field-change field-descriptor answer evt)
```

Comment:
Jos tekee alla kuvatun `:on-change` muutoksen, niin tästä pitäis voida poistaa `@verify-email?` haara.